### DISCUSSION TITLE
Is there a way to "favorite" colors? I only see a "recent" option.

### DISCUSSION THREAD

**Discussion by adamfreeplaneuser01** [freeplane/freeplane#1492] [Help]
I would love to be able to save colors that I use often, but I am only seeing a "recent" tab.
<img width="526" alt="sdfaksdjfsdjsdf" src="https://github.com/freeplane/freeplane/assets/147225942/8a811a04-448e-4367-960b-5d7a8f75d247">

**Comment by dpolivaev** [comment]
Unfortunately, the component does not provide any reasonable way to access its internal state.
